### PR TITLE
Convert mistral workflows to orquesta

### LINKED ISSUE #66: NAPALM workflows need converting from mistral to Orquesta
Mistral-v2 runner has been removed from the master branch, and will not be available in StackStorm 3.3.0 and above.

This pack has some mistral-v2 workflows that need updating to Orquesta workflows:

- interface_down_workflow
- configuration_change_workflow

### PR DESCRIPTION
**Pull request by floatingstatic**
This closes #66 which I ran into installing the 0.4.1 version of this napalm pack in a fresh stackstorm v3.3.0 installation.

It appears as though the `check_consistency_chatops` workflow was added in #17 with nothing referencing it (no metadata file). Oddly it contains the same workflow tasks as `configuration_change_workflow` so I am opting to just remove the file.

### CODE REVIEW

**Comment by floatingstatic** [comment]
There appears to be some issue with the ci test environment related to the version of the cryptography module as part of `st2common==3.4.dev0`:

```
Installed /home/circleci/virtualenv/lib/python3.6/site-packages/eventlet-0.25.1-py3.6.egg
error: cryptography 2.8 is installed but cryptography>=3.2 is required by {'pyOpenSSL'}
make: *** [Makefile:273: .install-runners] Error 1
Original script exit code: 2

Exited with code exit status 2
CircleCI received exit code 2
```

I'm not sure how to proceed with fixing this.

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## 0.4.2
+
+- Convert mistral workflows to orquesta
```
Comment (arm4b) [inline CHANGES.md:5]: Because it's not just a bugfix patch, can we bump the pack version to `0.5.0` instead?

**Comment by arm4b** [comment]
Yeah, we're updating cryptography dependency in st2 core. Once the https://github.com/StackStorm/st2/pull/5095 is merged, this PR CI should be :heavy_check_mark: .

**Comment by arm4b** [comment]
@floatingstatic Thanks for the PR!
Are you folks heavily relying on this pack in production?
If you're interested, we could give your team maintainer access to this pack in Exchange.

File: CHANGES.md (hunk 1 above)
Comment (floatingstatic) [reply CHANGES.md:5]: Of course. Done!

**Comment by floatingstatic** [comment]
> @floatingstatic Thanks for the PR!
> Are you folks heavily relying on this pack in production?
> If you're interested, we could give your team maintainer access to this pack in Exchange.

I use st2 internally at work but all of the packs are custom (we do not use NAPALM). I do however use this pack at home for personal use.  If you are looking for some help maintaining it I would be happy to assist if there is a need here. Let me know.

**Comment by arm4b** [comment]
@floatingstatic Thanks! Created https://github.com/StackStorm-Exchange/stackstorm-napalm/pull/68 for that.

**Review by arm4b — APPROVED** [review]
Thanks for the contribution :+1:

**Pull request merged**